mito-ai/src/Extensions/AiChat/CTACarousel.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { JupyterFrontEnd } from '@jupyterlab/application';
+import { OperatingSystem } from '../../utils/user';
+import { COMMAND_MITO_AI_SETTINGS } from '../SettingsManager/SettingsManagerPlugin';
+import '../../../style/CTACarousel.css';
+
+interface CTACarouselProps {
+    app: JupyterFrontEnd;
+    operatingSystem: OperatingSystem;
+}
+
+const CTACarousel: React.FC<CTACarouselProps> = ({ app, operatingSystem }) => {
+    const [currentIndex, setCurrentIndex] = useState(0);
+
+    // Define messages inside component to access app prop
+    const CTACAROUSEL_MESSAGES = [
+        {
+            content: (
```

Comment:
This is sweet. Once change here is could we add a consistent height to the carrasouel content so switching from the first one to the other is not so abrupt. If we just hardcode a heigh, I think it would fix this. 

https://github.com/user-attachments/assets/ff008d94-5cc9-420f-9879-c20f7b8354d9